runtime/lang/menu_ja_jp.utf-8.vim

```diff
@@ -1,315 +1,1266 @@
-" vi:set ts=8 sts=8 sw=8 tw=0:
-"
-" Menu Translations:	Japanese (UTF-8)
-" Last Translator:	MURAOKA Taro  <koron.kaoriya@gmail.com>
-" Last Change:		20-Feb-2022.
-"
-" Copyright (C) 2001-2022 MURAOKA Taro <koron.kaoriya@gmail.com>,
-"			  vim-jp <http://vim-jp.org/>
+" Menu Translations:	  Japanese
+" Maintainer:           Ada (Haowen) Yu <me@yuhaowen.com>
+" Previous Maintainer:  MURAOKA Taro  <koron.kaoriya@gmail.com>
+" Last Change:          2022 July 10
+" Original translations
 "
-" THIS FILE IS DISTRIBUTED UNDER THE VIM LICENSE.
+" Generated with the scripts from:
 "
-" Original translations
+"       https://github.com/adaext/vim-menutrans-helper
 
 " Quit when menu translations have already been done.
+
 if exists("did_menu_trans")
   finish
 endif
 let did_menu_trans = 1
-let s:keepcpo= &cpo
+let s:keepcpo = &cpo
 set cpo&vim
 
 scriptencoding utf-8
 
 " Help menu
-menutrans &Help			ヘルプ(&H)
-menutrans &Overview<Tab><F1>	概略(&O)<Tab><F1>
-menutrans &User\ Manual		ユーザーマニュアル(&U)
-menutrans &How-To\ links	&How-toリンク
-menutrans &Credits		クレジット(&C)
-menutrans Co&pying		著作権情報(&P)
-menutrans &Sponsor/Register	スポンサー/登録(&S)
-menutrans O&rphans		孤児(&R)
-menutrans &Version		バージョン情報(&V)
-menutrans &About		Vimについて(&A)
-
-let g:menutrans_help_dialog = "ヘルプを検索したいコマンドもしくは単語を入力してください:\n\n挿入モードのコマンドには i_ を先頭に付加します. (例: i_CTRL-X)\nコマンドライン編集コマンドには c_ を先頭に付加します. (例: c_<Del>)\nオプションの名前には ' を付加します. (例: 'shiftwidth')"
+menutrans &Help ヘルプ(&H)
```

Comment:
I prefer using tab(s) instead of a space between the original message (i.e. "&Help" in this case) and the translated message ("ヘルプ(&H)" in this case).
Using only one space is hard to distinguish the original part and the translated part.

Reply:
Thanks. I've replace it. Also, I've add a new commit which have been modified according to your comments.